src/lib/components/ILSFooter/ILSFooter.js

```diff
@@ -1,4 +1,4 @@
-import { getStaticPageByName } from '@config/uiConfig';
+import { getStaticPageByName, uiConfig } from '@config/uiConfig';
```

Comment:
nit: maybe the `uiConfig` should be imported from `@config` as everywhere else.

Reply:
if we abuse too much of these collective index exports it might lead to circular dependencies and if we have a habit of importing from exact location it is less likely to happen